dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/TrackedEntityAttributeController.java

```diff
@@ -223,4 +189,22 @@
         return result;
     }
 
+    private TrackedEntityAttribute getTrackedEntityAttribute( String id )
+        throws WebMessageException
+    {
+        TrackedEntityAttribute trackedEntityAttribute = trackedEntityAttributeService.getTrackedEntityAttribute( id );
+
+        if ( trackedEntityAttribute == null )
+        {
+            throw new WebMessageException( WebMessageUtils.notFound( TrackedEntityAttribute.class, id ) );
+        }
+
+        if ( trackedEntityAttribute.getTextPattern() == null )
+        {
+            throw new WebMessageException( WebMessageUtils.badRequest( "Attribute does not contain pattern." ) );
```

Comment:
badRequest and probably notFound are already statically imported, so you can remove `WebMessageUtils.`